src/internal/connector/sharepoint/list.go

```diff
@@ -182,8 +177,8 @@ func fetchColumns(
 			if resp.GetOdataNextLink() == nil {
 				break
 			}
-
-			builder = tc.NewColumnsRequestBuilder(*resp.GetOdataNextLink(), gs.Adapter())
+			// *mssite.SitesItemListsItemContentTypesItemColumnsRequestBuilder
```

Comment:
is this comment needed?

Reply:
No. It is not

Reply:
Resolved as of: https://github.com/alcionai/corso/pull/1479/commits/8a997b1a6c83f0b142a526a66385639e79a825ae